public/translations/en.json

```diff
@@ -51,6 +51,7 @@
   "selectLayersToView": "Select layers to view",
   "selectPoints": "Select Points",
   "shareUrl": "Share url",
+  "wmsUrl": "WMS url",
```

Comment:
There's no difference between the translation of "wmsUrl" in English and Dutch, is this field necessary?